cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_initial.spec.js

```diff
@@ -0,0 +1,150 @@
+    });
+
+    after(() => {
+      cy.deleteWorkspaceById(workspaceId);
+    });
+
+    it('should contain correct content', () => {
+      // contains initial page title and description
+      cy.contains('Welcome to OpenSearch').should('exist');
+      cy.contains('My workspaces').should('exist');
+      cy.contains(
+        'Collaborate on use-case based projects with workspaces. Select a workspace to get started.'
+      ).should('exist');
+
+      // contains five use case title
+      cy.contains('Observability').should('exist');
+      cy.contains('Security Analytics').should('exist');
+      cy.contains('Search').should('exist');
+      cy.contains('Essentials').should('exist');
+      cy.contains('Analytics').should('exist');
+
+      // contains no workspace message
+      cy.contains('No workspaces').should('exist');
+
+      // contains created workspaces
+      cy.contains(workspaceName).should('exist');
+
+      // contains correct link
+      cy.contains('a', 'Learn more from documentation').should('exist');
+      cy.get(
+        'a[href="https://opensearch.org/docs/latest/opensearch/index/"]'
+      ).should('have.attr', 'target', '_blank');
+      cy.contains(
+        'a',
+        'Explore live demo environment at playground.opensearch.org'
+      ).should('exist');
+      cy.get('a[href="https://playground.opensearch.org/"]').should(
+        'have.attr',
+        'target',
+        '_blank'
+      );
+
+      // contain left bottom button
+      cy.get('[id$="popoverForSettingsIcon"]').should('exist');
+      cy.getElementByTestId('openDevToolsModal').should('exist');
+      if (Cypress.env('SECURITY_ENABLED')) {
+        cy.getElementByTestId('account-popover').should('exist');
+      }
+    });
+
+    it('should show correct use case information', () => {
+      cy.getElementByTestId(
+        'workspace-initial-useCaseCard-observability-button-information'
+      ).click();
+      cy.contains('Use cases').should('exist');
+      cy.contains(
+        'Gain visibility into system health, performance, and reliability through monitoring of logs, metrics and traces.'
+      ).should('exist');
+    });
+
+    it('should navigate to the workspace', () => {
+      cy.contains(workspaceName).click();
+
+      cy.location('pathname', { timeout: 6000 }).should(
+        'include',
+        `/w/${workspaceId}/app/`
+      );
+    });
+
+    it('should navigate to workspace create page', () => {
+      cy.getElementByTestId(
+        'workspace-initial-card-createWorkspace-button'
+      ).click();
+
+      cy.getElementByTestId(
+        'workspace-initial-button-create-essentials-workspace'
+      ).click();
+      cy.location('pathname', { timeout: 6000 }).should(
+        'include',
+        `/app/workspace_create`
+      );
+      cy.location('hash').should('include', 'useCase=Essentials');
+
+      cy.getElementByTestId('workspaceUseCase-essentials').click({
+        force: true,
+      });
```

Comment:
Should it be an assertion that the use case is selected by default?

Reply:
Thanks, the assertion has been added.